Userland/Utilities/xzcat.cpp

```diff
@@ -22,7 +22,8 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     args_parser.parse(arguments);
 
     auto file = TRY(Core::File::open_file_or_standard_stream(filename, Core::File::OpenMode::Read));
-    auto stream = TRY(Compress::XzDecompressor::create(move(file)));
+    auto buffered_file = TRY(Core::BufferedFile::create(move(file)));
```

Comment:
For GZIP, we found it somewhat beneficial to increase the buffer size to 256KiB. You might want to play around with that as well.